users-service/src/repository/UsuarioRepository.js

```diff
@@ -0,0 +1,70 @@
+const Usuario = require('../models/Usuario');
+const Time = require('../models/Time');
+
+const RETORNO_SUCESSO = 1;
+const RETORNO_ERRO = 2;
+const RETORNO_ERRO_INESPERADO = 3;
+
+module.exports = {
+   async addUser(nome, cargo, tecnologias) {
+      const tecnologiasArray = tecnologias
+         .split(',')
+         .map(tecnologia => tecnologia.trim());
+
+      const usuario = await Usuario.create({
+         nome,
+         cargo,
+         tecnologias: tecnologiasArray,
+      });
+
+      return usuario;
+   },
+   async findUserByName(name) {
+      const usuario = await Usuario.find({ nome: name }, () => {});
+
+      return usuario;
+   },
+   async findUserById(id) {
+      const usuario = await Usuario.findById(id);
+
+      return usuario;
```

Comment:
Acho que é melhor já retornar direito, não?
O que vc acha?

```suggestion
      return await Usuario.findById(id);
```